app/lib/constituency_api.rb

```diff
@@ -1,25 +1,32 @@
 module ConstituencyApi
   ConstituencyApiError = Class.new(RuntimeError)
 
-  class Constituency < Struct.new(:name)
+  class Constituency < Struct.new(:name, :mp)
   end
-  
+
+  class Mp < Struct.new(:id, :name, :start_date)
+    URL = "http://www.parliament.uk/biographies/commons"
+    def url
+      "#{URL}/#{name.parameterize}/#{id}"
+    end
+  end
+
```

Comment:
Are these meant to be value objects? If so `Struct` may not be the best thing to inherit from as it generates `attribute=` methods as well. Also any missing initialize arguments are just nils.

A more explicit way of creating a value object would be something like this:

``` ruby
class Constituency
  attr_reader :name, :mp

  def initlalize(name, mp)
    @name, @mp = name, mp
  end

  def ==(other)
    other.is_a?(self.class) && name == other.name && mp == other.mp
  end
  alias_method :eql?, :==
end
```

This is for future guidance though - it's probably not worth making the change